README.md

```diff
@@ -36,7 +36,7 @@ of data.
 from gluonts.dataset.util import to_pandas
 from gluonts.dataset.pandas import PandasDataset
 from gluonts.dataset.repository.datasets import get_dataset
-from gluonts.model.deepar import DeepAREstimator
+from gluonts.mx import DeepAREstimator
```

Comment:
Let's maybe not change this in the README for now.

Users expect the example from the GitHub project page to work with the current version of GluonTS.